imports/server/migrations/38-consolidate-profiles.ts

```diff
@@ -0,0 +1,28 @@
+import { Meteor } from 'meteor/meteor';
+import { Mongo } from 'meteor/mongo';
+import { Migrations } from 'meteor/percolate:migrations';
+import * as t from 'io-ts';
+import { UserCodec } from '../../lib/schemas/user';
+
+// Since the profiles model has been removed, we need to make our own collection
+// for this migration.
+const Profiles = new Mongo.Collection<{ _id: string } & t.TypeOf<typeof UserCodec>['profile']>('jr_profiles');
+
+Migrations.add({
+  version: 38,
+  name: 'Consolidate profiles onto Meteor.users',
+  up() {
+    Meteor.users.find({ profile: { $ne: null as any } }).forEach((u) => {
+      Meteor.users.update(u._id, { $unset: { profile: 1 } }, { validate: false } as any);
+    });
+
+    Profiles.find({}).forEach((profile) => {
+      Meteor.users.update(profile._id, { $set: { profile } });
```

Comment:
If I'm not mistaken: this sets a field `_id` on the nested profile object?  Maybe pluck just the fields we actually expect here instead?